src/Test/Tasty/Plutarch/Property.hs

```diff
@@ -193,6 +196,61 @@ classifiedProperty getGen shr getOutcome classify comp = case cardinality @ix of
                                             Left e' -> failCrashyGetOutcome e'
                                             Right s' -> crashedWhenItShouldHave e s'
 
+{- | `alwaysfailproperty` operates similarly to `classifiedProperty` but optimized
+ for cases that will always fail. Scripts will only run once and will have some
+ considerable performance benefit as compared to `classifiedProperty`.
+-}
+alwaysFailProperty ::
```

Comment:
I don't think this really makes sense as a conditional property. If you expect it to _always_ fail, then it's universal IMHO.